python/lsst/pipe/tasks/make_psf_matched_warp.py

```diff
@@ -60,6 +62,12 @@ class MakePsfMatchedWarpConnections(
     },
 ):
     """Connections for MakePsfMatchedWarpTask"""
+    sky_map = Input(
```

Comment:
@erykoff, want to weigh in on what to call this connection?  I hate the proliferation of "sky" "map" variants, but I think you're less resigned to it than I am...

Reply:
Once @erykoff weighs in on `^[sS]ky[_]?[mM]ap$` this PR/ticket is good to go.

Reply:
Thanks, Eli! This is also consistent with other usage of `sky_map` connection, mostly introduced by Eli elsewhere. For e.g., https://github.com/lsst/pipe_tasks/blob/e7c99ddb59f0c91ad7ed5a058e0712b776b49d51/python/lsst/pipe/tasks/healSparseMapping.py#L353